HydroModelBuilder/ModelInterface/flopyInterface/viz/MT3D_PP_viz.py

```diff
@@ -189,24 +190,24 @@ def viewConcsByZone(self, nper='all', specimen=None):
     modelmap.plot_bc('RIV', plotAll=True)
     try:
         modelmap.plot_bc('WEL', plotAll=True)
-    except:
+    except Exception:
         pass
     modelmap.plot_bc('GHB', plotAll=True)
     modelmap.plot_bc('SFR', plotAll=True)
     try:
         modelmap.plot_bc('DRN', plotAll=True)
-    except:
+    except Exception:
         pass
     ax.axes.xaxis.set_ticklabels([])
 
     ax = fig.add_subplot(2, 4, 2, aspect='equal')
     ax.set_title('Coonambidgal')
     modelmap = flopy.plot.ModelMap(model=self.mf_model.mf)
     min_conc = -100.0
-    max_conc = 100. #1E6
+    max_conc = 100.0
     temp = max_conc
```

Comment:
This temp variable is never used, but it looks like it might be important

Reply:
@ConnectedSystems , Ok I think some of these plotting functions that are quite specific to the Campaspe need to live in the Campaspe repo and make use of the flopy object. I'll look to do that soon.